tests/test.cpp

```diff
@@ -151,8 +151,37 @@ flatbuffers::unique_ptr_t CreateFlatBufferTest(std::string &buffer) {
 void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length) {
 
   // First, verify the buffers integrity (optional)
-  flatbuffers::Verifier verifier(flatbuf, length);
-  TEST_EQ(VerifyMonsterBuffer(verifier), true);
+  std::vector<uint8_t> test_buff;
+  test_buff.resize(length * 4 + 5);
```

Comment:
This all seems very arbitrary. All we need is a vector, then append the flatbuffer twice, then check only that the computed length == original length.